src/hooks/useWorkletStateMachine/index.ts

```diff
@@ -1,4 +1,3 @@
-import fastMerge from 'expensify-common/dist/fastMerge';
```

Comment:
Is this change related?

Reply:
I just realized that it originates from merge conflicts from another PR. Reverted now